app/models/rule_event.rb

```diff
@@ -1,15 +1,17 @@
 class RuleEvent
   def self.load(attributes)
     object = allocate
+    object.instance_variable_set(:@identifier, attributes.fetch("identifier", ""))
```

Comment:
This line ensures we do not bust on existing events that do not have an identifier yet.

Reply:
```suggestion
    # Ensures we do not bust on historic events that do not have an identifier
    object.instance_variable_set(:@identifier, attributes.fetch("identifier", ""))
```